packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js

```diff
@@ -42,7 +42,9 @@ function TaxonomiesPanel(props) {
 
   const availableTaxonomies = taxonomies.filter((taxonomy) => {
     const isVisible = taxonomy?.visibility?.show_ui;
-    const canAssignTerms = Boolean(capabilities[`assign-${taxonomy?.slug}`]);
+    const canAssignTerms =
+      Boolean(capabilities[`assign-${taxonomy?.slug}`]) ||
+      Boolean(capabilities[`assign-${taxonomy?.restBase}`]);
```

Comment:
Do we want to check each individually or just default to restBase first and then `slug`? so

```js
Boolean(capabilities[`create-${taxonomy.slug}`]) || Boolean(capabilities[`create-${taxonomy?.restBase}`])
```
vs

```js
Boolean(capabilities[`assign-${taxonomy?.restBase || taxonomy?.slug}`])
```

In my mind this is a little sturdier in case the permission matches the slug rather than the rest base but either way works for me 🤔